seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  res.status(200).json({ data: rows });
+});
+
+app.delete("/postsdelete", async (req, res) => {
+  const { postId } = req.body;
+  await appDataSource.query(
+    `DELETE
+    FROM posts
+    WHERE posts.id = ${postId}
+    `
+  );
```

Comment:
```suggestion
  await appDataSource.query(
    `DELETE
    FROM posts
    WHERE posts.id = ?
    `, [ postId ]
  );
```
템플릿 리터럴(`${}`)보다 place holder(`?`)를 사용할 수 있도록 해주세요.